chrome/_locales/pt_BR/messages.json

```diff
@@ -0,0 +1,28 @@
+{
+    "extDescription": {
+      "message": "Bloquear anúncios nas transmissões da Twitch.tv.",
+      "description": "A descrição da extensão."
+    },
+    "toggleAdblocker": {
+        "message": "Bloquear anúncios"
+    },
+    "toggleBlockingAds": {
+        "message": "Aviso de \"Bloqueando anúncios\"",
+        "description": "Ativar ou desativar a mensagem 'Bloqueando anúncios...'"
+    },
+    "reloadTabsWarning": {
+        "message": "Sempre que você alterar uma configuração da extensão, você vai precisar recarregar a(s) guia(s) da Twitch."
```

Comment:
Could be simplified to:
"Sempre que alterar uma configuração, você precisa de recarregar a(s) guia(s) da Twitch.".

Also, does the term "guia" actually used daily to refer to browser tabs? From experience, the loan word "tabs" is the most used term in PT